src/webui/www/private/scripts/client.js

```diff
@@ -292,6 +292,15 @@ window.addEvent('load', function() {
         $('desktopFooterWrapper').addClass('invisible');
     }
 
+    // Show Filters Sidebar is enabled by default
+    let showFiltersSidebar = (LocalPreferences.get('show_filters_sidebar') === null)
```

Comment:
The value is now stored in a variable and reused addressing [your issue](https://github.com/qbittorrent/qBittorrent/pull/15409#discussion_r700728813).
The function is still called multiple times but it now follows the same pattern of the other options.

Reply:
>The value is now stored in a variable and reused addressing your issue.

Seriously you did not address any of my comments at all... Let's try one last time:
1. Use `const showFiltersSidebar` here and create another variable in the click event handler.
2. assign `LocalPreferences.get('show_filters_sidebar')` to a local variable to avoid repetitive calls.
3. Use a helper function to get `LocalPreferences.get('show_filters_sidebar')`, that would ensure you will handle empty, invalid values consistently.

Reply:
Done, let me know if there is anything else to fix.